Rubberduck.UnitTesting/UnitTesting/TestResult.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Rubberduck.UnitTesting
 {
-    public class TestResult
+    public struct TestResult
```

Comment:
Why not `readonly struct`?

Reply:
Because I let R# do that and the IDE complains that it's a 7.2 feature?  :-P

Will fix - it indeed should be readonly.